README.md

```diff
@@ -11,7 +11,7 @@ Requirements
 Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
 
 ## ansible.cfg
-This role is designed to work with merge "hash_behaviour". Make sure your
+This role is designed to work with merge "hash\_behaviour". Make sure your
```

Comment:
Why the extra slash ?

Reply:
It's a little pedantic but you can include double underscores to make something bold in markdown and the slash escapes to be an explicit underscore.